src/main/telemetry/crsf.c

```diff
@@ -284,9 +284,7 @@ static uint8_t crsfSchedule[CRSF_SCHEDULE_COUNT_MAX];
 
 #if defined(USE_MSP_OVER_TELEMETRY)
 void scheduleMspResponse() {
-    if (!mspReplyPending) {
-        mspReplyPending = true;
-    }
+    mspReplyPending = true;
```

Comment:
Good one, @codecae. I assumed that the early exit was required for CRSF to allow requests to be queued up.